examples/with-apollo-and-redux/lib/initApollo.js

```diff
@@ -0,0 +1,36 @@
+import { ApolloClient, createNetworkInterface } from 'react-apollo'
+import fetch from 'isomorphic-fetch'
+
+let apolloClient = null
+
+// Polyfill fetch() on the server (used by apollo-client)
+if (!process.browser) {
```

Comment:
This part is not necessary as this is something ["isomorphic-fetch" already does](https://github.com/matthew-andrews/isomorphic-fetch#warnings):

> This adds fetch as a global so that its API is consistent between client and server.